profileTemplate

```diff
@@ -36,18 +36,17 @@ $mail_user      = "yourname\@example.com";      # who will be notified
 $get_dicom_info = "/PATH/TO/get_dicom_info.pl"; # path to get_dicom_info.pl
 $horizontalPics = 1;
 $no_nii         = 0;                            # 0=create nifti files, 1=don't create nifti files
+$PatientID_regex  = '([^_]+)_(\d+)_([^_]+)'       # regular expression to parse PatientName or PatientID
```

Comment:
haha. Ooops. I just realized I made a stupid comment. ;) Never mind.

Sorry about that.